eXist-db/exist
# Upgrade yajsw to v12.11

Tested:
- run `java -jar start.jar`
- run  `./tools/yajsw/bin/runConsole.sh `
- install run remove deamon
 -- `sh ./tools/yajsw/bin/installDaemon.sh`
 -- `sh ./tools/yajsw/bin/queryDaemon.sh`
 -- `sh ./tools/yajsw/bin/stopDaemon.sh`
 -- `sh ./tools/yajsw/bin/uninstallDaemon.sh`

> this release removes log entries where it complains about missing optional jar files

```
yajsw-stable-12.11

    * Bug: windows: wrapper.affinity should support up to 64 processors
    * Bug: wrong yajsw version logged

yajsw-stable-12.10

    * New: wrapper.ping.check_ack property for bidirectional ping check
    * New: wrapper.delay_shutdown property. Delay shutdown to allow scripts, such as emails to terminate.
    * Bug: bad manifest
    * Bug: windows: wrapper.affinity should support up to 64 processors
    * Bug: NPE when executing shell script
    * Bug: Posix: cannot start process if wrapper.java.additional contains blank
    * Bug: Cannot start jboss wildfily since 12.09: groovy-patch: disable java logging.
    * Change: do not show messages "INFO: lib not found" for extended libs.
```
